modules/gentemplates/openapi-generator/src/main/java/com/reprezen/genflow/openapi/generator/OagCodegenGenTemplateBase.java

```diff
@@ -64,6 +64,32 @@ public void configure() throws GenerationException {
 				"Each property should be a JSON object with a name/value pair for each property.",
 				"Example: for '-Dmodels -Dapis=User,Pets' use the following:", "value:", "  models: ''",
 				"  apis: Users,Pets"));
+		define(parameter().named(LANGUAGE_SPECIFIC_PRIMITIVES).optional().withDescription(
+				"Specifies additional language specific primitive types.",
+				"You can have multiple occurrences of this option.", //
+				"Example: -Pet", //
+				"         -User"));
+		define(parameter().named(TYPE_MAPPINGS).optional().withDescription(
+				"Sets mappings between Swagger spec. types and generated code types.", //
```

Comment:
A few issues:
* We should replace "Swagger spec. types" with "standard OpenAPI types".  
* Aside from `array` and `map`, what are the standard types that can be specified as keys here? It would be much easier for our users to know exactly what those types are. I assume there are not many of them. If you can easily get this information from the SCG or OAG docs or source code, it would be nice to have that here, but it's not worth spending a lot of time looking for it. 
* It would be helpful to clarify that this refers to the _declared type_ in the generated code, as distinct from `INSTANTIATION_TYPES` which refers to the actual runtime type.

So I'd rewrite this as follows:
> "Specifies mappings between standard OpenAPI types and declared types in generated code.",
> "The standard OpenAPI types include map, array, ..."

That second line should only be included if we find the definitive answer as to what standard types are allowed here. And in that case please include the full list.

Reply:
> Aside from array and map, what are the standard types that can be specified as keys here? It would be much easier for our users to know exactly what those types are. I assume there are not many of them. If you can easily get this information from the SCG or OAG docs or source code, it would be nice to have that here

The following data-types are supported - `string, number, integer, boolean, array, object`. For the first four, the SCG/OAG defaults to - `String, BigDecimal, Integer, Boolean, List` respectively. 
Note - Type mappings (for example - integer: AtomicInteger, array: Set) can be added to alter the generated code. 
 
> It would be helpful to clarify that this refers to the declared type in the generated code, as distinct from INSTANTIATION_TYPES which refers to the actual runtime type.

This is only applicable to arrays (array: Set, array: List) where you generally bind to an interface. For primitive types, the generated declared type is a runtime type (Integer, BigDecimal, AtomicInteger...).

Reply:
> "Specifies mappings between standard OpenAPI types and declared types in the generated code.",
"The standard OpenAPI types include map, array, ..."

Shouldn't we use "standard types" or "standard OpenAPI Codegen types" in place of "Standard OpenAPI types"? I believe the later will lead to confusion as OpenAPI has [no data-type named "map"](https://github.com/OAI/OpenAPI-Specification/issues/38#issuecomment-56212819), like "array". SCG/OAG has support for using "map" keyword - `map: HashMap` which it translates correctly in the generated code.